Src/UIComponents/ProfileInfo/ProfileInfo.ts

```diff
@@ -0,0 +1,51 @@
+import tile from '../TileComponent/TileComponent';
+import {userIcon} from './User';
+const moment = require('moment');
+
+function createStyledElement(tagName: string, classNames: string[], innerHTML?: string): HTMLElement {
+    const element = document.createElement(tagName);
+    if(classNames.length) element.classList.add(...classNames);
+    if(innerHTML) element.innerHTML = innerHTML;
+    return element;
+}
+
+function profileInfo(profileData: {
+    username: string,
+    gender: 'Male' | 'Female',
+    dateOfBirth: Date,
+    height: number, // cm
+    weight: number, // kg
+    activityLevel: 'Low' | 'Regular' | 'Active' | 'Very Active',
```

Comment:
Hey, here is an enum, you could use to indicate options:
https://github.com/KrystianKjjk/CodersCamp2020.Project.TypeScript.FitNotFat/blob/IP_Models/Models/ActivityLevel.model.ts

Reply:
Good point. Your suggestion is applied.
If you don't have any additional suggestions, you may merge this branch to master.